src/Home.js

```diff
@@ -1,37 +1,50 @@
 import React, { Component } from 'react';
 
-import {HomePageIconList} from 'interra-data-catalog-components'
-import {Hero} from 'interra-data-catalog-components'
-
-
-const axios = require('axios');
+import HomePageIconList from './components/HomePageIconList';
+import { Hero } from 'interra-data-catalog-components';
+import backend from './services/backend';
+import Loader from 'react-loader';
 
 class Home extends Component {
 
-    state = {
-        "items": [],
-        "state": "loading"
-    }
-
-    componentDidMount() {
-        axios.get(`http://dkan.local/api/v1/organization`)
-            .then(res => {
-                const items = res.data.map(x => {
-                    var item = {identifier: x}
-                    return item
-                });
-                this.setState({ "items": items, "state": "ok" });
-            });
-    }
-
-    render() {
-        return (
-          <div>
-            <Hero/>
-            <HomePageIconList state={ this.state.state } items={ this.state.items } />
-          </div>
-        );
-    }
+	state = {
+    "items": [],
+		"loaded": false
+	}
+
+  async fetchData() {
+		const { data } = await backend.get(`/collections/organization.json`);
```

Comment:
We need to use the dkan_api endpoints instead of the interra_api endpoints.

Reply:
Why would we use different endpoints? The point here is we are not tied to DKAN. We are already implementing the Interra endpoints. We could update them or make them more compatible with data packages if that would work better. Lets talk on Tues.